lib/administrate/search.rb

```diff
@@ -31,6 +38,28 @@ def search_attributes
       end.keys
     end
 
-    attr_reader :resolver, :term
+    def search_scope(term)
+      if term && (term[-1, 1] == ":")
+        possible_scope = term[0..-2]
+        possible_scope if resource_class.respond_to?(possible_scope) &&
+                          !banged?(possible_scope) &&
```

Comment:
Use 4 (not 18) spaces for indenting a condition in an `if` statement spanning multiple lines.